src/astro/sql/operators/sql_dataframe.py

```diff
@@ -1,35 +1,20 @@
-import inspect
-from typing import Dict, Optional
+from typing import Dict
 
-import pandas as pd
 from airflow.decorators.base import DecoratedOperator
-from airflow.providers.sqlite.hooks.sqlite import SqliteHook
 
-from astro.constants import Database
 from astro.databases import create_database
 from astro.settings import SCHEMA
 from astro.sql.table import Table
-from astro.utils import get_hook
-from astro.utils.database import create_database_from_conn_id
-from astro.utils.dependencies import (
-    BigQueryHook,
-    PostgresHook,
-    SnowflakeHook,
-    postgres_sql,
-)
-from astro.utils.load import load_dataframe_into_sql_table
+from astro.utils.dataframe_function_handler import DataframeFunctionHandler
 from astro.utils.table_handler import TableHandler
 
 
-class SqlDataframeOperator(DecoratedOperator, TableHandler):
+class SqlDataframeOperator(DataframeFunctionHandler, DecoratedOperator, TableHandler):
```

Comment:
Since this already lives within the `sql` folder, perhaps we could call this `DataframeOperator` only, and the module could be named `dataframe.py`